### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by ijeomaodinko**
**Requirements**
Create a method called current_user in your ApplicationController to make current user data available to all controllers.
It will return the first user from the database.
Don't worry about proper authentication at this time. You will add it in the upcoming projects.
Create forms to perform the following functions:
Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
Allow Users to add likes to Posts.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
   get 'users/show'
+  get 'posts/new'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 #  resources :users
   # Defines the root path route ("/")
   # root "articles#index"
   root "users#index"
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show] 
+    resources :posts, only: [:index, :new, :create, :show] do
```
Comment (codepantha) [inline config/routes.rb]: - [ ] An error occurs when I try to create a post as shown below and the post never gets created. As a result, I can't really test the comment and likes functionalities. Please fix this bug 🐛 and the next reviewer will test the comment and test functionalities 👍 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @ijeomaodinko 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] I noticed that the link to the `MIT` license in the README file is broken. Please fix it 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ijeomaodinko** [comment]
Add the required changes

**Review by danielufeli — APPROVED** [review]
Hi @ijeomaodinko ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ChopApproveCopApproveGIF

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**